pkg/systemd/init.js

```diff
@@ -218,6 +227,31 @@ define([
                 });
         }
 
+        function wait_valid(proxy, callback, args) {
+            proxy.wait(function() {
+            if (proxy.valid)
+                callback(proxy, args);
+            });
+        }
+
+        function add_timer_properties(timer_unit,unit) {
+            unit.LastTriggerTime = moment(timer_unit.LastTriggerUSec/1000).fromNow();
+            if (timer_unit.LastTriggerUSec === 0)
+                unit.LastTriggerTime = _("not known");
+            var next_run_time = 0;
```

Comment:
you're mixing different capitalization styles (`LastTriggerTime`, `next_run_time`)
You can either stick to the old one (`next_run_time`), since this is an edit of an existing file, or write the new code with camelCase.